runners/flink/job-server/build.gradle

```diff
@@ -40,15 +40,15 @@ apply plugin: "application"
 mainClassName = "org.apache.beam.runners.flink.FlinkJobServerDriver"
```

Comment:
we should be able to drop the evaluationDependsOn above because it was moved into the task.

Reply:
Makes sense.